nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/download/DownloadServiceImpl.java

```diff
@@ -105,6 +106,8 @@
 
     private static final String RUN_FUNCTION = "run";
 
+    private static final Pattern FILENAME_SANITIZATION_REGEX = Pattern.compile(";\\w+=[^?]*$");
```

Comment:
I still don't see what the `[^?]` has to do here. If it's important then add a test showing where it's interesting. If it's not then replace it with something simpler.

Reply:
agree, let's go with a simpler `.*`